### ISSUE TITLE
Linters fail on master branch

### ISSUE THREAD

**Issue by ivankorn** [terraform-google-modules/terraform-google-vpc-service-controls#8]
```
[17:49][user@host:~/workspace/google/terraform-google-vpc-service-controls]$ make -s
Running shellcheck
Running flake8
./test/verify_boilerplate.py:60:15: N803 argument name 'ARGS' should be lowercase
./test/verify_boilerplate.py:188:28: N803 argument name 'ARGS' should be lowercase
Running go fmt and go vet
Running terraform validate
terraform validate --check-variables=false . 
terraform validate --check-variables=false ./examples/simple_example 
terraform validate --check-variables=false ./examples/simple_example_access_level 
terraform validate --check-variables=false ./examples/simple_example_bridge 
terraform validate --check-variables=false ./modules/access_level 
terraform validate --check-variables=false ./modules/bridge_service_perimeter 
terraform validate --check-variables=false ./modules/regular_service_perimeter 
terraform validate --check-variables=false ./test/fixtures/simple_example 
Running hadolint on Dockerfiles
Checking for required files LICENSE README.md
Testing the validity of the header check
..
----------------------------------------------------------------------
Ran 2 tests in 0.008s

OK
Checking file headers
Checking for trailing whitespace
./modules/access_level/README.md:5:## Usage 
./modules/bridge_service_perimeter/README.md:5:## Usage 
./modules/regular_service_perimeter/README.md:5:## Usage 
./.kitchen/logs/kitchen.log:4:I, [2019-07-24T14:34:04.390992 #2784]  INFO -- Kitchen: 
make: *** [Makefile:62: check_trailing_whitespace] Error 1
[17:49][user@host:~/workspace/google/terraform-google-vpc-service-controls]$ echo $?
2
[17:50][user@host:~/workspace/google/terraform-google-vpc-service-controls]$ 
```

```
[17:50][user@host:~/workspace/google/terraform-google-vpc-service-controls]$ terraform -v
Terraform v0.11.14

Your version of Terraform is out of date! The latest version
is 0.12.5. You can update by downloading from www.terraform.io/downloads.html
[17:52][user@host:~/workspace/google/terraform-google-vpc-service-controls]$ git status
On branch master
Your branch is up to date with 'origin/master'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	credentials.json.ikorniienko
	credentials.json_ivankorniienko

nothing added to commit but untracked files present (use "git add" to track)
[17:52][user@host:~/workspace/google/terraform-google-vpc-service-controls]$ 
```

**Comment by aaron-lane** [comment]
Fixed by #9.

**Comment by ivankorn** [comment]
> Fixed by #9.
White spaces. 
Flake8 fixed [here](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/pull/10/files#diff-fe6d6ed874db1cc4d1455b5f7299c393R60-R283) in #10
